config/dev.exs

```diff
@@ -56,6 +56,9 @@ config :companies, CompaniesWeb.Endpoint,
     ]
   ]
 
+# Do not run appsignal in local dev
+config :appsignal, :config, active: false
```

Comment:
I know this is technically out of scope, but without this I accidentally got some errors submitted to my work AppSignal account, because I guess I set the env var for APPSIGNAL_PUSH_API_KEY at some point and it was still set... very confusing seeing errors from an unrelated project show up in work project 🙃